### DISCUSSION TITLE
possible to use OW-parameter for detection?

### DISCUSSION THREAD

**Discussion by vasmue** [AntSimi/py-eddy-tracker#116] [Q&A]
Hi,

I would like to use py-eddy-tracker to identify and track eddies in a high-resolution Arctic model simulation. Unfortunately the SSH signal of eddies in the Arctic tends to be quite weak (especially under ice) and the main eddy signal is often somewhere below the surface.
Would it be possible (and feasible) to "trick" the algorithm similar to the LAVD detection example (https://py-eddy-tracker.readthedocs.io/en/latest/python_module/07_cube_manipulation/pet_lavd_detection.html), so that the threshold contours of Okubo-Weiss parameter are used for eddy detection instead of SSH contours?

Thanks in advance for any ideas

**Comment by AntSimi** [comment]
Hi,
Currently there is no example in gallery.
A git user forked py-eddy-tracker to use OW field to detect eddies : [fork](https://github.com/Mesharou/py-eddy-tracker).
I start an example to detect eddies with OW, but i need more work to release it, i hope soon.

**Reply by vasmue** [reply]
Thank you Antoine! That's very helpful, I will have a look at the forked code.
